openmc/mgxs/__init__.py

```diff
@@ -42,3 +44,409 @@
+    6.309573e-2, 6.606934e-2, 6.918310e-2, 7.244360e-2, 7.585776e-2,
+    7.943282e-2, 8.317638e-2, 8.709636e-2, 9.120108e-2, 9.549926e-2,
+    1.000000e-1, 1.047129e-1, 1.096478e-1, 1.148154e-1, 1.202264e-1,
+    1.258925e-1, 1.318257e-1, 1.380384e-1, 1.445440e-1, 1.513561e-1,
+    1.584893e-1, 1.659587e-1, 1.737801e-1, 1.819701e-1, 1.905461e-1,
+    1.995262e-1, 2.089296e-1, 2.187762e-1, 2.290868e-1, 2.398833e-1,
+    2.511886e-1, 2.630268e-1, 2.754229e-1, 2.884032e-1, 3.019952e-1,
+    3.162278e-1, 3.311311e-1, 3.467369e-1, 3.630781e-1, 3.801894e-1,
+    3.981072e-1, 4.168694e-1, 4.365158e-1, 4.570882e-1, 4.786301e-1,
+    5.011872e-1, 5.248075e-1, 5.495409e-1, 5.754399e-1, 6.025596e-1,
+    6.309573e-1, 6.606934e-1, 6.918310e-1, 7.244360e-1, 7.585776e-1,
+    7.943282e-1, 8.317638e-1, 8.709636e-1, 9.120108e-1, 9.549926e-1,
+    1.000000, 1.047129, 1.096478, 1.148154, 1.202264,
+    1.258925, 1.318257, 1.380384, 1.445440, 1.513561,
+    1.584893, 1.659587, 1.737801, 1.819701, 1.905461,
+    1.995262, 2.089296, 2.187762, 2.290868, 2.398833,
+    2.511886, 2.630268, 2.754229, 2.884032, 3.019952,
+    3.162278, 3.311311, 3.467369, 3.630781, 3.801894,
+    3.981072, 4.168694, 4.365158, 4.570882, 4.786301,
+    5.011872, 5.248075, 5.495409, 5.754399, 6.025596,
+    6.309573, 6.606934, 6.918310, 7.244360, 7.585776,
+    7.943282, 8.317638, 8.709636, 9.120108, 9.549926,
+    1.000000e1, 1.020000e1, 1.040000e1, 1.060000e1, 1.080000e1,
+    1.100000e1, 1.120000e1, 1.140000e1, 1.160000e1, 1.180000e1,
+    1.200000e1, 1.220000e1, 1.240000e1, 1.260000e1, 1.280000e1,
+    1.300000e1, 1.320000e1, 1.340000e1, 1.360000e1, 1.380000e1,
+    1.400000e1, 1.420000e1, 1.440000e1, 1.460000e1, 1.480000e1,
+    1.500000e1, 1.520000e1, 1.540000e1, 1.560000e1, 1.580000e1,
+    1.600000e1, 1.620000e1, 1.640000e1, 1.660000e1, 1.680000e1,
+    1.700000e1, 1.720000e1, 1.740000e1, 1.760000e1, 1.780000e1,
+    1.800000e1, 1.820000e1, 1.840000e1, 1.860000e1, 1.880000e1,
+    1.900000e1, 1.920000e1, 1.940000e1, 1.960000e1, 1.980000e1,
+    2.000000e1, 2.100000e1, 2.200000e1, 2.300000e1, 2.400000e1,
+    2.500000e1, 2.600000e1, 2.700000e1, 2.800000e1, 2.900000e1,
+    3.000000e1, 3.200000e1, 3.400000e1, 3.600000e1, 3.800000e1,
+    4.000000e1, 4.200000e1, 4.400000e1, 4.600000e1, 4.800000e1,
+    5.000000e1, 5.200000e1, 5.400000e1, 5.600000e1, 5.800000e1,
+    6.000000e1, 6.500000e1, 7.000000e1, 7.500000e1, 8.000000e1,
+    9.000000e1, 1.000000e2, 1.100000e2, 1.200000e2, 1.300000e2,
+    1.400000e2, 1.500000e2, 1.600000e2, 1.800000e2, 2.000000e2,
+    2.400000e2, 2.800000e2, 3.200000e2, 3.600000e2, 4.000000e2,
+    4.400000e2, 4.800000e2, 5.200000e2, 5.600000e2, 6.000000e2,
+    6.400000e2, 6.800000e2, 7.200000e2, 7.600000e2, 8.000000e2,
+    8.400000e2, 8.800000e2, 9.200000e2, 9.600000e2, 1.e3])
+GROUP_STRUCTURES['UKAEA-1102'] = np.array([
+    0., 1.0000e-11, 1.0471e-11, 1.0965e-11, 1.1482e-11, 1.2023e-11, 1.2589e-11,
```

Comment:
Where is this structure in the reference?

Also, same off by factor of 1e6 comment, and maybe the 0 and 1e-11 comment

Reply:
Corrected factor of 1e6 and added a [reference](_UKAEA-1102 https://fispact.ukaea.uk/wiki/UKAEA-1102_group_structure)